src/main/java/com/coveros/selenified/element/Get.java

```diff
@@ -385,12 +379,10 @@ public int numOfTableColumns() {
      */
     public List<List<WebElement>> tableColumns() {
         if (!element.is().present()) {
-            return null; // NOSONAR - returning an empty array could be confused
-            // with no rows
+            return null; // returning an empty array could be confused with no rows
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS1168)